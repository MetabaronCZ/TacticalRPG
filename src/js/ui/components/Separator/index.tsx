import React from 'react';

const Separator = (): JSX.Element => (
	<hr className="Separator" />
);

export default Separator;
