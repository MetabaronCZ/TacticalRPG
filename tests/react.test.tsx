import * as React from 'react';
import { shallow } from 'enzyme';

import Separator from 'ui/common/Separator';

describe('React component test example', () => {
	test('#Separator', () => {
		const sep = shallow(<Separator />);
		expect(sep).toMatchSnapshot();
		expect(sep.hasClass('Separator')).toBeTruthy();
	});
});
