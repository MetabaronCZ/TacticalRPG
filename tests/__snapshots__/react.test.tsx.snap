// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React component test example #Separator 1`] = `ShallowWrapper {}`;
